### DISCUSSION TITLE
How to disable multiple object paths creation.

### DISCUSSION THREAD

**Discussion by joseigor** [Kistler-Group/sdbus-cpp#298] [Q&A]
Hi all, 

I am facing the following issue and I would appreciate it if someone could help me to understand how to solve it.

When I create an object using the following string  as object path `/org/sdbuscpp/concatenator` I see in D-Feet that the following paths are created:
- `/`
- `/org/`
- `/org/sdbuscpp/`
- `/org/sdbuscpp/concatenator`

Is there a way to just create the path -`/org/sdbuscpp/concatenator` ?

I am following the same example as in the guide:
```c
// Create D-Bus connection to the system bus and requests name on it.
    const char* serviceName = "org.sdbuscpp.concatenator";
    auto connection = sdbus::createSystemBusConnection(serviceName);

    // Create concatenator D-Bus object.
    const char* objectPath = "/org/sdbuscpp/concatenator";
    Concatenator concatenator(*connection, objectPath);
```

Thanks in advance!!!

**Comment by sangelovic** [comment]
I don't know :) Have you found more information on that in the meantime?
